Flickr/Sources/FlickrAppVCs/PersistenceManager.swift

```diff
@@ -6,46 +6,43 @@
 //
 
 import Foundation
+import GRDB
 
 class PersistenceManager {
-    static func storeData(_ favoritesData: FavoriteImagesArray?) {
-        let searchDirectory = FileManager.SearchPathDirectory.documentDirectory
-        guard let url = FileManager.default.urls(for: searchDirectory, in: .userDomainMask).first?.appendingPathComponent(fileName, isDirectory: false) else {
-            return
-        }
+    static func retrieveData(dbPool: DatabasePool) -> [FavoriteImageData]? {
+        var favoriteimagesArray: [FavoriteImageData] = []
         do {
-            let data = try JSONEncoder().encode(favoritesData)
-            if FileManager.default.fileExists(atPath: url.path) {
-                try FileManager.default.removeItem(at: url)
```

Comment:
Please keep the purpose of persistence manager to be same as demonstrated in Meditation App.